runtime_scan/pkg/scanner/job_managment.go

```diff
@@ -86,16 +90,19 @@ func (s *Scanner) jobBatchManagement(scanDone chan struct{}) {
 		}(data, s.killSignal)
 	}
 
+	// wait for killSignal or fullScanDone
 	select {
 	case <-s.killSignal:
 		log.WithFields(s.logFields).Info("Scan process was canceled")
 	case <-fullScanDone:
 		log.WithFields(s.logFields).Infof("All jobs has finished")
 		// Nonblocking notification of a finished scan
+		// notify back to server on scan done
```

Comment:
job management shouldn't be aware that there is a server.

Reply:
he is not, the comment meant to help the maintainers of the code to understand better

Reply:
not a good thing to put comments in an infra code about specific implementation of users.